Source/ASTableView.mm

```diff
@@ -1969,4 +1969,11 @@ - (void)didMoveToSuperview
   }
 }
 
+#pragma mark - Accessibility overrides
+
+- (NSArray *)accessibilityElements {
+  [self waitUntilAllUpdatesAreCommitted];
```

Comment:
I’m not familiar with accessbility and not sure when this method is called, but are you sure we want to block the (main?) thread in this case? Would it be better to somehow trigger a accessibility elements recollection once the table/collection finishes its updates.